sass/_variables.scss

```diff
@@ -27,11 +26,10 @@ $small-icon-size: 16px;
     transition: pointer-events 0.2s cubic-bezier(0.64, 0.04, 0.35, 1), transform 0.25s cubic-bezier(0.64, 0.04, 0.35, 1), opacity 0.17s cubic-bezier(0.64, 0.04, 0.35, 1), z-index 0.17s cubic-bezier(0.64, 0.04, 0.35, 1);
 }
 
-@mixin svg-icon-color($color) {
+@mixin svg-icon-color($color, $important:false) {
```

Comment:
do you really need this `$important`? You only use this once. I think you can do without it.